guidelines.mdx

```diff
@@ -0,0 +1,517 @@
+---
+meta:
+  title: Scaleway Documentation Guidelines
+  description: Discover comprehensive guidelines on writing and formatting for our documentation site. Ensure consistency and clarity with our style guide, covering grammar, terminology, and best practices.
+content:
+  h1: Scaleway Documentation Guidelines
+  paragraph: Discover comprehensive guidelines on writing and formatting for our documentation site. Ensure consistency and clarity with our style guide, covering grammar, terminology, and best practices.
+tags: style-guide guidelines wording grammar
```

Comment:
Should we add validation/posted date?

Reply:
Validation date might be useful to showcase the document itself isn't outdated.
Posted may not be necessary, I think.